src/Prima/Pyxis/Form/Autocomplete.elm

```diff
@@ -18,7 +19,7 @@ module Prima.Pyxis.Form.Autocomplete exposing
 
 ## Configuration Methods
 
-@docs autocomplete, init, update, autocompleteChoice
+@docs autocomplete, init, update, autocompleteChoice, withThreshold, withDebouncer
```

Comment:
Queste funzioni sono nel posto sbagliato, vanno mantenute nel blocco `Options` in ordine alfabetico

Reply:
Penso il problema di quelle funzioni sia di origine diversa. Ciò che si trova nelle "Options" si occupa di aggiornare "AutocompleteOption", mentre quelle 2 funzioni applicano modifiche a livello di model, così da avere tali dati a disposizione durante l'update. Oltre al riordinamento alfabetico, cos'altro devo modificare?

Reply:
Secondo me va bene lo stesso spostare sotto il blocco Options dato che opzioni lo sono per davvero. Che poi la firma si  leggermente diversa dalle altre fn è un problema minore ;)